lib/serial_terminal.pm

```diff
@@ -385,13 +385,23 @@ sub select_serial_terminal {
         $console = 'root-ssh';
     } elsif (($backend eq 'generalhw' && !has_serial_over_ssh) || $backend eq 's390x') {
         $console = $root ? 'root-console' : 'user-console';
+    } elsif (check_pvm){
+        if (check_var('VIRTIO_CONSOLE', 0)) {
+            $console = $root ? 'root-console' : 'user-console';
+        } else {
+            $console = $root ? 'root-virtio-terminal' : 'user-virtio-terminal';
+        }
```

Comment:
could be

```perl
my $user = $root ? 'root' : 'user';
$console = check_var('VIRTIO_CONSOLE', 0) ? "$user-console" : "$user-virtio-terminal";
```

Reply:
could be also `$backend eq 'qemu' || check_pvm`